Makefile

```diff
@@ -70,16 +83,21 @@ vendor:
 	@echo "✅ vendor"
 .PHONY: vendor
 
-run: $(OPERATOR_SDK) gen
-	$(OPERATOR_SDK) run --local --operator-flags "operator run"
-.PHONY: run
+run: gen
+	go build -o $(NOOBAA_OPERATOR_LOCAL) -mod=vendor $(CMD_MANAGER)
+	$(BIN)/noobaa-operator-local operator run
+	.PHONY: run
```

Comment:
Apologies for this. I will submit a PR for correcting misaligned .PHONY. We can correct this here only if you agree